secrets/templates/idds.yaml

```diff
@@ -66,7 +66,7 @@ stringData:
               },
           {{- range $client := .Values.auth.oidc.clients }}
           {{$client.name | quote }}{{":"}}
-            {{- $.Files.Get "files/panda_auths/auth_config.json.template" | replace "ID" $client.client_id | replace "SECRET" $client.client_secret | replace "URL" $.Values.auth.oidc.config_url | replace "AUDIENCE" $.Values.auth.oidc.audience  | replace "NAME" $client.name }}
+            {{- $.Files.Get "files/panda_auths/auth_config.json.template" | replace "ID" $client.client_id | replace "SECRET" $client.client_secret | replace "CONF" $.Values.auth.oidc.config_url | replace "AUDIENCE" $.Values.auth.oidc.audience  | replace "NAME" $client.name }}
```

Comment:
This change is inconsistent with

https://github.com/PanDAWMS/panda-k8s/blob/main/secrets/files/panda_auths/auth_config.json.template

Reply:
I see. I changed it back.